source/extensions/filters/http/dynamic_modules/abi_impl.cc

```diff
@@ -34,6 +36,16 @@ size_t getHeaderValueImpl(const Http::HeaderMap* map,
   return values.size();
 }
 
+bool headerAsRequestAttribute(const Http::HeaderMap* map, const Envoy::Http::LowerCaseString header,
+                              envoy_dynamic_module_type_buffer_envoy_ptr* result_buffer_ptr,
+                              size_t* result_buffer_length_ptr) {
+  auto lower_header = header.get();
```

Comment:
can you check map is non-null? If the previous filter running before dynamic_modules does the local reply, then the request header phase will be called directly hence this can be null pointer.

Reply:
Just to make sure I understand your non-null request correctly, are you asking that I add another check for null before calling `getHeaderValueImpl` which is already doing a null check on the map?